### YamlMime:ManagedReference
items:
- uid: system.fabric.exception._fabric_invalid_address_exception
  id: _fabric_invalid_address_exception
  parent: system.fabric.exception
  children:
  - system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException()
  - system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String)
  - system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String,Throwable)
  - system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(Throwable)
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException
  nameWithType: FabricInvalidAddressException
  fullName: system.fabric.exception.FabricInvalidAddressException
  type: Class
  source:
    remote: &o0
      path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
      branch: master
      repo: https://msazure.visualstudio.com/One/_git/sf-java
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 5
  package: system.fabric.exception
  summary: <p>The exception that is thrown when an address is not recognized by Service fabric. </p>
  syntax: &o1
    content: public class FabricInvalidAddressException extends FabricException
  inheritance:
  - java.lang.Object
  - RuntimeException
  - system.fabric.exception._fabric_exception
  inheritedMembers:
  - system.fabric.exception._fabric_exception.FabricException()
  - system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String)
  - system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(Throwable)
  - system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  - system.fabric.exception._fabric_exception.getErrorCode()
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException()
  id: FabricInvalidAddressException()
  parent: system.fabric.exception._fabric_invalid_address_exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException()
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException()
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException()
  overload: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 13
  package: system.fabric.exception
  summary: <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_invalid_address_exception" data-throw-if-not-resolved="false">FabricInvalidAddressException</xref> class. </p>
  syntax:
    content: public FabricInvalidAddressException()
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String)
  id: FabricInvalidAddressException(String)
  parent: system.fabric.exception._fabric_invalid_address_exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException(String message)
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException(String message)
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException(String message)
  overload: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 40
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_invalid_address_exception" data-throw-if-not-resolved="false">FabricInvalidAddressException</xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.</p>

    <p></p>
  syntax:
    content: public FabricInvalidAddressException(String message)
    parameters:
    - id: message
      type: "26831127"
      description: <p>Error message that explains the reason for the exception. </p>
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String,Throwable)
  id: FabricInvalidAddressException(String,Throwable)
  parent: system.fabric.exception._fabric_invalid_address_exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException(String message, Throwable cause)
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException(String message, Throwable cause)
  overload: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 28
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_invalid_address_exception" data-throw-if-not-resolved="false">FabricInvalidAddressException</xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.</p>

    <p></p>
  syntax:
    content: public FabricInvalidAddressException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
      description: <p>Error message that explains the reason for the exception. </p>
    - id: cause
      type: a04fafd7
      description: <p>Exception that is the cause of the current exception, or a null reference if no inner exception is specified. </p>
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(String,Throwable,boolean,boolean)
  id: FabricInvalidAddressException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_invalid_address_exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  overload: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 48
  package: system.fabric.exception
  syntax:
    content: protected FabricInvalidAddressException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException(Throwable)
  id: FabricInvalidAddressException(Throwable)
  parent: system.fabric.exception._fabric_invalid_address_exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  langs:
  - java
  name: FabricInvalidAddressException(Throwable cause)
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException(Throwable cause)
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException(Throwable cause)
  overload: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricInvalidAddressException.java
    startLine: 44
  package: system.fabric.exception
  syntax:
    content: public FabricInvalidAddressException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
references:
- uid: system.fabric.exception._fabric_invalid_address_exception.FabricInvalidAddressException*
  name: FabricInvalidAddressException
  nameWithType: FabricInvalidAddressException.FabricInvalidAddressException
  fullName: system.fabric.exception.FabricInvalidAddressException.FabricInvalidAddressException
  package: system.fabric.exception
- uid: "26831127"
  spec.java:
  - name: String
    fullName: String
- uid: a04fafd7
  spec.java:
  - name: Throwable
    fullName: Throwable
- uid: 4fc6e284
  spec.java:
  - name: boolean
    fullName: boolean
- uid: system.fabric.exception._fabric_exception.FabricException()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException()
  nameWithType: FabricException.FabricException()
  fullName: system.fabric.exception.FabricException.FabricException()
  type: Constructor
  syntax:
    content: public FabricException()
- uid: system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(FabricErrorCode errorCode)
    parameters:
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message)
  nameWithType: FabricException.FabricException(String message)
  fullName: system.fabric.exception.FabricException.FabricException(String message)
  type: Constructor
  syntax:
    content: public FabricException(String message)
    parameters:
    - id: message
      type: "26831127"
- uid: system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause)
  nameWithType: FabricException.FabricException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause)
  nameWithType: FabricException.FabricException(Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.getErrorCode()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: getErrorCode()
  nameWithType: FabricException.getErrorCode()
  fullName: FabricErrorCode system.fabric.exception.FabricException.getErrorCode()
  type: Method
  syntax:
    content: public FabricErrorCode getErrorCode()
    return:
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException
  nameWithType: FabricException
  fullName: system.fabric.exception.FabricException
  type: Class
  syntax:
    content: public class FabricException
- uid: system.fabric.exception._fabric_invalid_address_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_invalid_address_exception.yml
  name: FabricInvalidAddressException
  nameWithType: FabricInvalidAddressException
  fullName: system.fabric.exception.FabricInvalidAddressException
  type: Class
  summary: <p>The exception that is thrown when an address is not recognized by Service fabric. </p>
  syntax: *o1
